app/decorators/article_decorator.rb

```diff
@@ -1,6 +1,15 @@
 class ArticleDecorator < ApplicationDecorator
   LONG_MARKDOWN_THRESHOLD = 900
 
+  def data_info_as_json(style: "full")
+    {
+      id: user_id,
+      className: "User",
+      style: style,
+      name: cached_user.name
+    }.to_json
+  end
```

Comment:
Refactor (blocking): I'm not sure why this is causing the name property once rendered as JSON to be surrounded by double quotes twice, but it's causing a JSON.parse error in application code which is why E2E tests are failing. The other weird thing is it appears to be only happening for the logged in user if an article of theirs is rendered.

![image](https://user-images.githubusercontent.com/833231/148452122-2544daba-3ff3-4867-996a-d2e183d71f8b.png)

See 

`data-info='<%= story.data_info_as_json %>'>` down further in _single_story.html.erb

Reply:
Oh…it's plenty weird.  And I can't duplicate what you're seeing.  @nickytonline could we pair on Friday to look further into this?